iati/fetch/tests/test_dataset.py

```diff
@@ -72,3 +73,39 @@ def test_get_dataset_correctly_encoded(self):
     def test_get_dataset_bad_url(self):
         """Given an invalid dataset URL, an Exception is raised."""
         pass
+
+    def test_get_publishers(self):
+        """Result of iati.fetch.get_publishers() will be a list and contain expected results."""
+        mock_publishers_response = pkg_resources.resource_stream(__name__, 'organization_list.json').read().decode('utf-8')
+        mock_publishers_url = 'http://test.com/publishers'
```

Comment:
Have a look at changes to the TestDataset tests and extracting the mock URL to keep things DRY.